src/fragments/start/getting-started/reactnative/setup.mdx

```diff
@@ -1,58 +1,209 @@
+? Distribution Directory Path: web-build
+? Build Command:  npx expo export:web
+? Start Command: npx expo start
+Using default provider  awscloudformation
+? Select the authentication method you want to use: AWS profile
+? Please choose the profile you want to use default
+
+...
+```
+</Block>
+
+<Block name="React Native CLI">
+
+```console
+
+? Enter a name for the project (amplifiedtodo) 
+The following configuration will be applied:
+
+Project information
+| Name: amplifiedtodo
+| Environment: dev
+| Default editor: Visual Studio Code
+| App type: javascript
+| Javascript framework: react-native
+| Source Directory Path: src
+| Distribution Directory Path: /
+| Build Command: npm run-script build
+| Start Command: npm run-script start
+
+? Initialize the project with the above configuration? Yes
+Using default provider  awscloudformation
+? Select the authentication method you want to use: AWS profile
+? Please choose the profile you want to use default
+
+...
 ```
 
 </Block>
 </BlockSwitcher>
 
-## Install Amplify libraries
+> Amplify CLI will infer the proper configuration based on the type of project Amplify is being initialized in where possible. Here it detected Create React App and provided the proper configuration for type of app, framework, source, distribution, build, and start options.
 
-Next, install the local Amplify dependencies. The directions here will depend on whether you are using Expo or the React Native CLI.
+When you initialize a new Amplify project, a few things happen:
 
-import all0 from "/src/fragments/start/getting-started/reactnative/getting-started-steps-ui.mdx";
+- It creates a top level directory called **amplify** that stores your backend definition. During the tutorial you'll add capabilities such as a GraphQL API and authentication. As you add features, the **amplify** folder will grow with infrastructure-as-code templates that define your backend stack. Infrastructure-as-code is a best practice way to create a replicable backend stack.
+- It creates a file called **aws-exports.js** in the **src** directory that holds all the configuration for the services you create with Amplify. This is how the Amplify client is able to get the necessary information about your backend services.
+- It modifies the **.gitignore** file, adding some generated files to the ignore list
+- A cloud project is created for you in the AWS Amplify Console that can be accessed by running `amplify console`. The Console provides a list of backend environments, deep links to provisioned resources per Amplify category, status of recent deployments, and instructions on how to promote, clone, pull, and delete backend resources
 
-<Fragments fragments={{all: all0}} />
+## Install Amplify Libraries
 
-Now your project is set up and you can begin adding new features.
+The `aws-amplify` package is the main library for working with Amplify Libraries in your projects:
 
-## Run your app
+```bash
+npm install aws-amplify
+```
```

Comment:
Was the original fragment outdated for this? It's my understanding that there are a lot of other dependencies (E.g. AsyncStorage) that need to be installed as well. I think the original fragment also included a block switcher that included a pod install step.

Reply:
The original tutorial imports from `aws-amplify` before installing it, and then suggests `npx expo start` before `amplify init`. A lot of issues reported in general about RN docs.

So I started based on React tutorial.  The idea is to test everything with `expo`. If everything works, test everything with RN and android. And I hope not to be the only one working on it, so someone who has more experience can iron out the details. For everything iOS I will have to guess/copy/assume.